tests/unit/testConfigs/bucketNotifConfigTest.js

```diff
@@ -0,0 +1,142 @@
+const assert = require('assert');
+const { bucketNotifAssert } = require('../../../lib/Config');
+
+describe('bucketNotifAssert', () => {
+    it('should not throw an error if bucket notification config is valid', () => {
+        assert.doesNotThrow(() => {
```

Comment:
https://nodejs.org/api/assert.html#assert_assert_doesnotthrow_fn_error_message doesNotThrow doesn't really do anything as stated in the docs. If you call a method and everything is peachy then the test passes. If it fails the test will fail. No, assert required. 
Note that the extra parameter of error message is also redundant with the test name that is displayed in the logs anyway.

Reply:
I can remove the `doesNotThrow`, but see my below response about the error messages